index.js

```diff
@@ -40,7 +41,10 @@ for (const key in commandFiles) {
 
     const command = new (require(`./src/Command/${file}`));
     command.initialize(dependencyGraph);
-    commandList.add(key, command);
+
+    /* Add to the command container */
+    commandList.add(command.commandName, command);
+    command.commandAliases.forEach(alias => commandList.add(alias, command));
```

Comment:
This is the fastest way to access a command alias. I did think about storing some form of list for each command, but that increases search times. The only downside here is that `!commands` now also shows all aliases, but maybe this isn't even a bad thing.

Reply:
I'll make an alias list in that case.